.travis.yml

```diff
@@ -9,4 +9,4 @@ env:
     - PINS="rrdd-plugins:."
     - BASE_REMOTE="https://github.com/xapi-project/xs-opam.git"
   matrix:
-    - DISTRO="debian-9-ocaml-4.07"
+    - DISTRO="debian-10-ocaml-4.08"
```

Comment:
i'll have a go at fixing this properly, so that we don't have to specify the ocaml version in this travis file